aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <cstring>
+
+namespace at {
+namespace vec256 {
+
+// NOTE: If you specialize on a type, you must define all operations!
+
+// emulates vectorized types
+template <class T>
+struct Vec256 {
+  T values[32 / sizeof(T)];
+  Vec256() {}
+  Vec256(T val) {
+    for (int i = 0; i != size(); i++) {
+      values[i] = val;
+    }
+  }
+  inline void load(const T *ptr) {
+    std::memcpy(values, ptr, 32);
+  };
+  static Vec256 s_load(const T* ptr) {
```

Comment:
What does the s_ stand for?

Reply:
`static`. I'm not tied to the name. You just can't have a static overload with the same name.